drivers/input/Kconfig.mcux_kpp

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config INPUT_MCUX_KPP
+	bool "KPP driver"
+	default y
+	depends on DT_HAS_NXP_MCUX_KPP_ENABLED
+	help
+	  Enable the driver of keypad port.
+
+if INPUT_MCUX_KPP
+config INPUT_KPP_PERIOD_MS
+	int "Sample period"
+	default 10
+	help
+	  Sample period in milliseconds when the key is pressed.
+endif # INPUT_MCUX_KPP
```

Comment:
normally for a single option we'd do `depends on INPUT_MCUX_KPP` rather than a whole `if/endif` block